### DISCUSSION TITLE
word:true without the trailing space?

### DISCUSSION THREAD

**Discussion by fharper** [espanso/espanso#1407] [Q&A] [answered]
I would like my trigger `bs` to work only when I type `bs` and not, as an example, `obs`. I can do nearly that with `word:true` but now I need to do `bs` and hit space.

I know, not a big deal to now have to type the spacebar after the trigger, but I've been using many shortcuts like that (without the space) for years (before I move to espanso), but thought I would find a way to prevent the shortcut to be triggered with other letters, which is why I found `word:true`, but it's not exactly what I need.

Any idea how can I do that?

Let me know if I'm not clear!

**Comment by kpym** [comment, marked as answer]
You can probably use `regex` in place of `trigger` like this
```yaml
matches:
  - regex: "\\bbs"
    replace: "Brothers & Sisters"
```
Here `\\b` is for "word boundary".

**Reply by fharper** [reply]
Sorry, totally missed this answer. That does the trick for the original question, thanks.

**Reply by smeech** [reply]
I agree, that's the only way of doing it if you don't know a space is going to prefix the `bs`.
By the way, if you dispense with the quote marks, you don't need to escape the "\\".
Are you happy (or for me) to close the discussion?

**Reply by fharper** [reply]
I just did, thanks.
